plugin.php

```diff
@@ -306,7 +306,7 @@ function rest_validate_request_arg( $value, $request, $param ) {
 		if ( isset( $args['format'] ) ) {
 			switch ( $args['format'] ) {
 				case 'date-time' :
-					if ( ! rest_parse_date( $value ) ) {
+					if ( $value && ! rest_parse_date( $value ) ) {
```

Comment:
I don't think we should necessarily only permit null values for dates, but not for other types or formats.

@WP-API/amigos Should we give `null` a free pass in validation (e.g. `null` is always an acceptable value for a field)?